src/popover/src/Popover.js

```diff
@@ -47,7 +47,7 @@ const Popover = memo(
         open,
         close
       }),
-      [popoverNode.current]
+      [open, close]
```

Comment:
Looks good to me (and makes sense - it seems the dependency arrays in that file need some work, but we can tackle those at another time.) Just a heads up, you might have better luck with this controlled behavior by using the `isShown` prop instead of grabbing the ref and calling `open`/`close` manually.

Reply:
Thanks, I'll take a look at `isShown`!